source/PHP/profile.php

```diff
@@ -32,16 +84,24 @@
       die();
     }
     //print out the info we obtained
-    echo "$user";
-    echo "$interests";
-    echo "$education";
-
+    echo "<div id=\"username\"><h1>$user</h1></div>";
+    echo "<div id=\"interests\">$interests</div>";
+    echo "<div id=\"education\">$education</div>";
   }
-  else
+
+  function map($user)
   {
-    //if a user name is not passed via GET, kill the page load and echo an error
-    echo "<h3>ERROR!</h3>";
-    die();
+    //code for google maps
+    echo "
+    <div id=\"map-canvas\" style=\"height:30%;margin-top:120px;\"></div>
```

Comment:
The client has requested that a users profile page has a map with all their posts on it